### PR TITLE
Task_06: Timers

### PR DESCRIPTION
**Pull request by denanko**
Module which works with timers using sysfs

### CODE REVIEW

File: 06_timers/userspace_script/show_time.c [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+#include <time.h>
+
+int main(void)
+{
+	struct timespec now = {0};
+
+	clock_gettime(0, &now);
+
+	printf("Absolute time is %ld.%ld seconds.\n", now.tv_sec, now.tv_nsec);
```
Comment (alexposukhov) [inline 06_timers/userspace_script/show_time.c]: Please convert this time to Human readable format.

**Review by alexposukhov — CHANGES_REQUESTED** [review]

**Comment by denanko** [comment]
show_time.c was updated with requested changes.

File: 06_timers/userspace_script/show_time.c [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	struct timespec now = {0};
+	struct tm *hrt = {0};		// human readable time
+
+	clock_gettime(0, &now);
+	hrt = localtime(&now.tv_sec);
+
+	printf("It is %d:%d:%d %d-%d-%d\n",
+			hrt->tm_hour, hrt->tm_min, hrt->tm_sec,
+			hrt->tm_year + 1900,
+			hrt->tm_mon, hrt->tm_mday);
```
Comment (alexposukhov) [inline 06_timers/userspace_script/show_time.c]: According to struct tm specification:
int tm_mon;         /* month, range 0 to 11             */
So in this case printf shows wrong month

**Review by alexposukhov — CHANGES_REQUESTED** [review]

File: 06_timers/userspace_script/show_time.c (hunk 2 above)
Comment (alexposukhov) [reply 06_timers/userspace_script/show_time.c]: Please add day of week

**Comment by denanko** [comment]
Month format fixed.

**Review by alexposukhov — APPROVED** [review]

**Pull request merged**